Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.java

```diff
@@ -278,6 +284,7 @@ private void addConfigSettings(ArrayList<SettingsItem> sl)
     sl.add(new SubmenuSetting(mContext, R.string.gamecube_submenu, MenuTag.CONFIG_GAME_CUBE));
     sl.add(new SubmenuSetting(mContext, R.string.wii_submenu, MenuTag.CONFIG_WII));
     sl.add(new SubmenuSetting(mContext, R.string.advanced_submenu, MenuTag.CONFIG_ADVANCED));
+    sl.add(new SubmenuSetting(mContext, R.string.tools_submenu, MenuTag.CONFIG_TOOLS));
```

Comment:
I think the new setting fits better in the Wii section personally.

Reply:
As long as you're ok with that, I'd do it. I just thought that the Wii section was specific to hardware "native" to it.

Reply:
I guess it is a Wii only piece of hardware

Reply:
It doesn't work with the GameCube, so to me it's Wii-specific :)